playbooks/common/openshift-cluster/upgrades/upgrade_nodes.yml

```diff
@@ -29,6 +29,10 @@
       {{ hostvars[groups.oo_first_master.0].openshift.common.admin_binary }} drain {{ openshift.node.nodename | lower }} --force --delete-local-data
     delegate_to: "{{ groups.oo_first_master.0 }}"
 
+  - name: Pause for pod rescheduling during node ugprade
+    pause:
+      seconds: "{{ openshift_node_upgrade_pause | default(30) | int }}"
+
```

Comment:
But the rescheduling happens immediately after you start the evacuation. In my test, which was only two nodes maybe it's different with more nodes, the pods were actually started on the other nodes before they'd fully terminated on the original node.